python/lsst/ap/association/diaPipe.py

```diff
@@ -202,6 +211,15 @@ class DiaPipelineConfig(pipeBase.PipelineTaskConfig,
         target=AssociationTask,
         doc="Task used to associate DiaSources with DiaObjects.",
     )
+    doSolarSystemAssociation = pexConfig.Field(
+        dtype=bool,
+        default=False,
+        doc="",
```

Comment:
While I think the name is pretty self-explanatory here please do give this a non-null docstring.